clang_format.py

```diff
@@ -20,17 +20,61 @@
 import tempfile
 import urllib.request
 from pathlib import Path
+from typing import Final, Literal, Mapping, Optional, Sequence, Tuple, Union
 
 # clang-format sha1s were retrieved at Chromium rev
 # 81cc23a856578b149a37dd109b147d8544f9cbd8.
 #
 # https://github.com/chromium/chromium/blob/master/buildtools/linux64/clang-format.sha1
 # https://github.com/chromium/chromium/blob/master/buildtools/mac/clang-format.sha1
 # https://github.com/chromium/chromium/blob/master/buildtools/win/clang-format.exe.sha1
-CLANG_FORMAT_SHAS = {
-    "Linux": "1baf0089e895c989a311b6a38ed94d0e8be4c0a7",
-    "Darwin": "62bde1baa7196ad9df969fc1f06b66360b1a927b",
-    "Windows": "d4afd4eba27022f5f6d518133aebde57281677c9",
+
+CLANG_FORMAT_SHAS: Final[
+    Mapping[
+        Tuple[int, int, int],
+        Mapping[Union[Literal["Linux"], Literal["Darwin"], Literal["Windows"]], str],
```

Comment:
No need for the `Union`: `Literal["Linux", "Darwin", "Windows"]`

Reply:
updated